timed/tracking/serializers.py

```diff
@@ -188,6 +190,87 @@ class Meta:
         ]
 
 
+class ReportIntersectionSerializer(Serializer):
+    """Serializers intersection of given reports."""
```

Comment:
I think this needs a bit more explanation. What exactly does this *do*? What *is* a report intersection?

Reply:
I added some more documentation which I hope makes things clearer. It should get even more clear in future PR where we will introduce bulk updates on reports.

The intersection is mainly needed for frontend to tell user which fields are already the same in given reports before bulk updating.